File: java-recipes/src/main/java/com/fargopolis/models/SubclassFeature.java
package com.fargopolis.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class SubclassFeature {
    private Integer subclassFeatureId;
    private Integer subclassId;
    private String name;
    private String description;
    private Integer level;
}

File: fargopolis-web/src/models/SubclassFeature.ts
export default class SubclassFeature {
    subclassFeatureId: number;
    name: string;
    level: number;
    description: string;
    subclassId: number;

    constructor(subclassFeatureId: number, name: string, level: number, description: string, subclassId: number) {
        this.subclassFeatureId = subclassFeatureId;
        this.name = name;
        this.level = level;
        this.description = description;
        this.subclassId = subclassId;
    }
}